azext_iot/monitor/handlers/handler.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.monitor.handlers import _internal
+from azext_iot.monitor.handlers.base_handler import AbstractBaseEventsHandler
+
+"""
+Use this handler if you aren't sure which handler is right for you
+"""
+
+
+class CommonHandler(AbstractBaseEventsHandler):
+    def __init__(
+        self,
+        device_id=None,
+        devices=None,
+        pnp_context=None,
+        interface_name=None,
+        content_type=None,
+        properties=None,
+        output=None,
+        validate_messages=None,
+        simulate_errors=None,
```

Comment:
The named args list here is getting too long - we should move the less commonly used to kwargs.

Reply:
Agreed, I'll be addressing this as part of the next work item where I think I'll split the central parsing logic out so we dont have the _static_validation or _dynamic_validation calls in the common parser